src/lib.rs

```diff
@@ -297,6 +297,7 @@ fn create_http_server(
   let server = HttpServer::new(move || {
     let cors_config = cors_config(&settings);
     let app = App::new()
+      .wrap(rate_limit_cell.message())
```

Comment:
Moved the message rate limit here so it doesnt have to be applied separately to many routes. This means a route like `POST /api/v3/comment` will have both message and post rate limits applied, but that shouldnt be a problem.